src/query/execute_statement.c

```diff
@@ -14217,93 +14248,24 @@ static PT_NODE *
 pt_cte_host_vars_index (PARSER_CONTEXT * parser, PT_NODE * node, void *arg, int *continue_walk)
 {
   PARSER_CONTEXT *query = (PARSER_CONTEXT *) arg;
```

Comment:
In the modified code, `query `is no longer used.

Reply:
good point. thanks.